src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -9542,4 +9542,108 @@ def test_aks_nodepool_update_with_nsg_control(self, resource_group, resource_gro
         cmd = 'aks delete --resource-group={resource_group} --name={name} --yes --no-wait'
         self.cmd(cmd, checks=[
             self.is_empty(),
-        ])    
\ No newline at end of file
+        ])
+
+    @live_only() # live only is required for test environment setup like `az login`
+    @AllowLargeResponse()
```

Comment:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=83547549&view=results) to validate the change.

Reply:
live test passed: https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=83605989&view=results